scripts/requestsSeed.js

```diff
@@ -0,0 +1,43 @@
+// ---------------- TEST DATA   ----------------
+// a page to input fake data in JSON FORMAT
+
+var mongoose = require("mongoose");
+const db = require("../models");
+
+mongoose.connect(
+  process.env.MONGODB_URI || "mongodb://localhost/neighborfavors_db"
+);
+
+const requestsSeed = [
+  {
+    client: {
```

Comment:
This is only what you do for the schema. You need to put valid mongoose object ids here for this to work